bedrock/base/templates/includes/protocol/footer/footer.html

```diff
@@ -63,12 +63,12 @@ <h5 class="mzp-c-footer-heading">
             <li><a href="{{ url('firefox.channel.desktop') }}#nightly" data-link-type="footer" data-link-name="Firefox Nightly">{{ ftl('footer-nightly') }}</a></li>
             <li><a href="{{ url('firefox.channel.android') }}#nightly" data-link-type="footer" data-link-name="Firefox Nightly for Android">{{ ftl('footer-nightly-for-android') }}</a></li>
             <li><a href="{{ url('firefox.enterprise.index') }}" data-link-type="footer" data-link-name="Firefox for Enterprise">{{ ftl('footer-enterprise') }}</a></li>
-            <li><a href="https://developer.mozilla.org/docs/Tools/?{{ utm_params }}&utm_content=developers" data-link-type="footer" data-link-name="Tools">{{ ftl('footer-tools') }}</a></li>
+            <li><a rel="external" href="https://firefox-source-docs.mozilla.org/devtools-user/?{{ utm_params }}&utm_content=developers" data-link-type="footer" data-link-name="Tools">{{ ftl('footer-tools') }}</a></li>
```

Comment:
For readability, it would be better to move `rel` behind `href`, or chronologically to the end:

```suggestion
            <li><a href="https://firefox-source-docs.mozilla.org/devtools-user/?{{ utm_params }}&utm_content=developers" data-link-type="footer" data-link-name="Tools" rel="external">{{ ftl('footer-tools') }}</a></li>
```

Reply:
Done. Thanks for the review.